apps/ios/src/index.tsx

```diff
@@ -2,11 +2,18 @@
 
 import {FluentTesterApp} from '@fluentui-react-native/tester';
 import * as React from 'react';
-import {AppRegistry, SafeAreaView} from 'react-native';
+import {AppRegistry, SafeAreaView, Text, useColorScheme} from 'react-native';
 
 const FluentTester: React.FunctionComponent = () => {
+  const isDarkMode = useColorScheme && useColorScheme() === 'dark';
+
+  const backgroundStyle = {
+    backgroundColor: isDarkMode ? 'black' : 'white',
+    flexGrow: 1,
```

Comment:
This addresses #623 , I need to see if we should include this fix in Android as well. Tagging @tamasane

Reply:
Nevermind. FIxed it separately in #630